src/bid/encoding.rs

```diff
@@ -160,22 +160,8 @@ impl StorageBid {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::score_gen::Score;
     use anyhow::Result;
-    use dusk_plonk::jubjub::{GENERATOR_EXTENDED, GENERATOR_NUMS_EXTENDED};
-    use rand_core::RngCore;
-
-    pub(self) fn gen_val_blinder_and_commitment(
-    ) -> (JubJubScalar, JubJubScalar, AffinePoint) {
-        let value = JubJubScalar::from(235_000u64);
-        let blinder = JubJubScalar::random(&mut rand::thread_rng());
-
-        let commitment: AffinePoint = AffinePoint::from(
-            &(GENERATOR_EXTENDED * value)
-                + &(GENERATOR_NUMS_EXTENDED * blinder),
-        );
-        (value, blinder, commitment)
-    }
```

Comment:
Why has been removed? This was helpful in order to set correct commitment values.

Reply:
The Perdersen commitment logic must be implemented as part of the bid structure, and not outside of it. There was a method `set_value` created for that

Reply:
Perfect, my bad.